Configure

```diff
@@ -1276,8 +1298,8 @@ if ($disabled{pic})
 	{
 	foreach (qw(shared_cflag shared_cxxflag shared_cppflag
                     shared_defines shared_includes shared_ldflag
-		    dso_cflags dso_cxxflags dso_cppflags
-                    dso_defines dso_includes dso_lflags))
+		    module_cflags module_cxxflags module_cppflags
```

Comment:
Inconsistent mix of tabs and spaces even here.

Reply:
Will fix